src/router/routes.ts

```diff
@@ -11,6 +11,11 @@ const routes: RouteRecordRaw[] = [
     component: () => import('layouts/LoginLayout.vue'),
     children: [{ path: '', component: () => import('pages/LoginPage.vue') }],
   },
+  {
+    path: '/register',
+    component: () => import('layouts/LoginLayout.vue'),
```

Comment:
If we use `LoginLayout` component for registration page component, layout component file and component name should be `RegisterLoginLayout.vue` and you can update component doc header, please.

https://github.com/auto-mat/ride-to-work-by-bike-frontend/blob/fd447cbacbbeac6f08fe421cdfee8800f35d73d6/src/layouts/LoginLayout.vue#L2-L26

Reply:
Absolutely. Updated.